prow/config/jobs/istio.yaml

```
@@ -250,6 +250,20 @@ jobs:
     - prow/config/ambient-sc.yaml
     - test.integration.ambient.kube
 
+  - name: integ-ambient-multicluster
```

```suggestion
  - name: integ-ambient-mc
```

Thoughts on using -mc instead of -multicluster? The other jobs use -mc instead of multicluster so this would be more consistent.

---

Yeah works for me